### PR TITLE
Blog slider alignment and filter issues fixes #81

### LINKED ISSUE #81: Implement the blog for the OpenEBS website
The blog for the OpenEBS website is to be implemented. This issue encompasses the following requirements.
- Implement the blog list section on the landing page
- Implement the blog page to list all the blogs
- Implement the pagination feature for the blog list
- Implement the blog content page
- Implement the list of similar blog posts
- Implement the filtering aspects based upon tags and authors

### PR DESCRIPTION
**Pull request by Pallavi-PH**
In this PR, We are fixing the following issues

1. Console error related to Asciinema
2. Handle blog slider when filtered blog length is 1 (While tag filtering)
Screenshot 2021-06-10 at 5 04 31 PM

3. Fix alignment issue in blog slider and tabs.
4. update styling of tabs and slider header
Screenshot 2021-06-10 at 5 00 48 PM
Screenshot 2021-06-10 at 5 01 08 PM

### CODE REVIEW

File: website/src/components/MiniBlog/styles.ts [hunk 1]
```diff
@@ -6,44 +6,55 @@ const useStyles = makeStyles((theme: Theme) => ({
...
     flexGrow: 1,
     fontWeight: 700,
     boxShadow: "2px 0px 33px 5px rgba(70, 68, 151, 0)",
   },
+  tabRoot: {
+    justifyContent: "center",
+    '& .Mui-selected': {
+      color: theme.palette.text.hint,
+      background: theme.palette.background.paper,
+      boxShadow: '2px 5px 31px 4px rgba(70, 68, 151, 0.07)',
```
Comment (IsAmrish) [inline website/src/components/MiniBlog/styles.ts:25]: should we try using color from palette?

File: website/src/components/MiniBlog/index.tsx [hunk 2]
```diff
@@ -188,13 +189,13 @@ const MiniBlog: React.FC = () => {
         </Container>
       </span>
       <Grid container justify="center">
-        <Grid item xs={10}>
+        <Grid item xs={filteredData.length === 1 ? 5 : 10}>
```
[the remark this answers is not in the document]
Comment (Pallavi-PH) [reply website/src/components/MiniBlog/index.tsx:192]: To align blog horizontally center if there is only one blog available for a particular filter.

File: website/src/components/MiniBlog/styles.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/components/MiniBlog/styles.ts:25]: The colors used in box shadows are unique. Not used elsewhere on the website. Currently, for all box-shadow values, we have directly included the rgba value. We can refractor all of them separately if required

Comment (IsAmrish) [reply website/src/components/MiniBlog/styles.ts:25]: it will be good to have all the colors in one place.

**Review by IsAmrish — APPROVED** [review]
lgtm

**Review by rakeshPRaghu — APPROVED** [review]
lgtm

File: website/src/components/MiniBlog/styles.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/components/MiniBlog/styles.ts:25]: Yes, will refractor it separately in another PR along with all other box-shadow colors

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**